addons/website/static/src/snippets/s_cta_badge/000.scss

```diff
@@ -0,0 +1,30 @@
+.s_cta_badge {
+    --s_cta_badge_link_margin: 20px;
+
+    margin: 1rem;
```

Comment:
I'd go with margin-bottom only, rationale:
The gap at the top is normally defined by the preceding element. If it's not enough or if there is no element preceding this snippet, then the user can add a new "line of text".
Similarly, left and right margins can be achieved by tapping "[space]" . 
These workarounds are not perfect but work and are intuitive for the user.
Enforcing a margin, instead,  may mismatch user needs and become  potentially problematic 😉